uniffi_bindgen/src/bindings/python/gen_python/mod.rs

```diff
@@ -437,6 +441,156 @@ impl PythonCodeOracle {
     }
 }
 
+impl VisitMut for PythonCodeOracle {
+    fn visit_record(&self, ci: &mut ComponentInterface) {
+        // Conversions for RecordTemplate.py
+        let mut new_records: BTreeMap<String, Record> = BTreeMap::new();
+
+        for (key, record_item) in ci.records.iter_mut() {
+            let mut record = record_item.clone();
+
+            // We just want to prefix reserved keywords for the name, without modifying it
+            record.rename(self.class_name(record_item.name()));
+
+            for field in &mut record.fields {
+                field.rename(self.var_name(field.name()));
```

Comment:
If I'm right about needing to rename type names, then I think you also would want to have a `field.rename_type()` method that called here.  This also applies to enum fields and function/method/constructor arguments.